### DISCUSSION TITLE
first mint image not showing up

### DISCUSSION THREAD

**Discussion by Zivsun12** [hashlips-lab/nft-erc721-collection#438] [Q&A]
Some issues when deploying
1. 1st mint does not pullup the Hidden gif
2. 2nd & on mints does Pull the Hidden gif
3. once I run reveal the images are kept as the hidden ones
4. next mint post reveal does bring up the correct NFT 
Any Advise?
https://testnets.opensea.io/collection/pp-gif-example

**Comment by liarco** [comment]
Hi @Zivsun12,
I already answered this questions on my [YouTube channel](https://www.youtube.com/watch?v=AFrsJRLrZEM&lc=Ugx6thVmULjdJxxgQSZ4AaABAg.9lTy4QesK4M9lU2AAev1Vr).

I hope it helps! ;)
